docs/components/clerk-provider.mdx

```diff
@@ -268,6 +268,13 @@ The `<ClerkProvider>` component must be added to your React entrypoint.
   - `InitialState`
 
   Provide an initial state of the Clerk client during server-side rendering. You don't need to set this value yourself unless you're [developing an SDK](/docs/references/sdk/overview).
+
```

Comment:
We should consider the docs at the top of the page with regards to dynamic rendering.


> By default, the <ClerkProvider> component has a server component wrapper in order to access header information. This means anything wrapped by the provider will be opted into dynamic rendering at request time.
>
> If you would prefer not to be opted into dynamic rendering, such as for static-optimized or ISR pages, you can either use <ClerkProvider> in a client component, or ensure the content is not wrapped by <ClerkProvider>. Instead of wrapping your application root with the provider, you can place it further down the tree to wrap route groups that require authentication.
>
> For example, if your project includes a set of landing/marketing pages and a dashboard that requires sign-in, you can create separate route groups for marketing and dashboard pages. By adding <ClerkProvider> to the (dashboard)/layout.jsx file, only the dashboard routes will opt into dynamic rendering, while the marketing routes remain statically optimized.